src/panels/lovelace/hui-root.ts

```diff
@@ -169,17 +172,50 @@ class HUIRoot extends LitElement {
                       ? ""
                       : html`
                           <mwc-list-item
+                            graphic="icon"
                             aria-label=${this.hass!.localize(
                               "ui.panel.lovelace.unused_entities.title"
                             )}
                             @request-selected="${this._handleUnusedEntities}"
                           >
+                            <ha-svg-icon slot="graphic"></ha-svg-icon>
                             ${this.hass!.localize(
                               "ui.panel.lovelace.unused_entities.title"
                             )}
                           </mwc-list-item>
                         `}
-                    <mwc-list-item @request-selected="${this._handleRawEditor}">
+                    <mwc-list-item
+                      graphic="icon"
+                      @request-selected="${this._handleManageDashboards}"
+                    >
+                      <ha-svg-icon
+                        slot="graphic"
+                        .path=${mdiViewDashboard}
+                      ></ha-svg-icon>
+                      ${this.hass!.localize(
+                        "ui.panel.lovelace.editor.menu.manage_dashboards"
+                      )}
+                    </mwc-list-item>
+                    <mwc-list-item
+                      graphic="icon"
+                      @request-selected="${this._handleManageResources}"
+                    >
+                      <ha-svg-icon
+                        slot="graphic"
+                        .path=${mdiViewDashboardOutline}
```

Comment:
We should use the same icon for resources everywhere, we now use a different icon in the config panels.

Reply:
For the "Manage dashboards" I took the icon from the config panels. The resource tab however does not have a separate icon so I had to choose one unless I missing a place where resources already get an icon?

Reply:
https://github.com/home-assistant/frontend/blob/dev/src/panels/config/lovelace/ha-config-lovelace.ts#L19

It is used on mobile